exp/constraints.lisp

```diff
@@ -388,35 +391,33 @@
   (begin (if-not-zero (min_cutoff_leq_16)
                       (begin (eq! (shift computation/RAW_ACC -1) (raw_lead_hi))
                              (eq! (shift computation/PLT_JMP -1) (min_cutoff)))
-                      (begin (if-zero (raw_hi_part_is_zero)
-                                      (begin (eq! (shift computation/RAW_ACC -1) (raw_lead_hi))
-                                             (eq! (shift computation/PLT_JMP -1) 16))
-                                      (begin (eq! (shift computation/RAW_ACC -1) (raw_lead_lo))
-                                             (eq! (shift computation/PLT_JMP -1) (- (min_cutoff) 16))))))))
+                      (begin (if-not-zero (raw_lead_hi_is_zero)
```

Comment:
Why are there 2 `(begin ...` ?

Reply:
We can `(force-bin (raw_lead_hi_is_zero))` to make it more efficient.

Reply:
There are two `begin` because one is the `if` case and the other is the `else` case